__tests__/integration/api.test.js

```diff
@@ -208,3 +211,84 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 responds with an array of an article's comments", () => {
+    const { commentData: testCommentData } = testData;
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        const expectedNumberOfComments = testCommentData.filter((comment) => {
+          return comment.article_id === 1;
+        }).length;
+        expect(comments.length).toBe(expectedNumberOfComments);
+      });
+  });
+
+  test("GET: 200 returns an empty array if article doesn't have any comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toEqual([]);
+      });
+  });
+
+  test("GET: 200 returned comments have the correct properties", () => {
+    const { commentData: testCommentData } = testData;
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        const expectedNumberOfComments = testCommentData.filter((comment) => {
+          return comment.article_id === 1;
+        }).length;
+        expect(comments.length).toBe(expectedNumberOfComments);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
You can be more exact about what you expect the `article_id` to be here, it should always be 1